localization/.backup/2016-02-20_02_13_01_1cac2917f3744cd9b527ba47556f05e8/localization.de.n3

```diff
@@ -0,0 +1,114 @@
+﻿# <auto-generated>
```

Comment:
The `.backup` directory is a local artifact and should be removed from the repo.